spec/lib/auth0/api/v2/jobs_spec.rb

```diff
@@ -24,7 +24,12 @@
     it { expect { @instance.import_users('', 'connnection_id') }.to raise_error('Must specify a valid file') }
     it { expect { @instance.import_users('users', '') }.to raise_error('Must specify a connection_id') }
   end
-
+  context '.export_users' do
+    it { expect(@instance).to respond_to(:export_users) }
+    it 'expect client to send post to /api/v2/jobs/users-exports' do
```

Comment:
This is testing the default options but there should also be one that tests passed-in parameters, making sure they get mapped correctly ... particularly `fields` since that's processed in the endpoint method.